config/purposes/ten_x.yml

```diff
@@ -0,0 +1,7 @@
+# # Test TenX plate purpose for SEQ-969 UAT only
```

Comment:
Should this file still be here?

Reply:
I'd like to keep that file there as a place holder for the actual 10x pipeline and for test purposes. Also it shows how the ten_stamp plate should be configured.

Reply:
This currently breaks the rake config:generate, which may break deployment. This is because 'YAML.parse_file' returns false which errors when you call '.to_ruby' on it.
Can be fixed by adding this line to top of file:

--- {} # Remove {} when enabling purpose below

Which makes the parse return something which '.to_ruby' can process.